src/graphql/employee.rs

```diff
@@ -0,0 +1,24 @@
+use crate::service::employee::update_password;
+
+pub struct EmployeeMutation;
+
+#[async_graphql::Object]
+impl EmployeeMutation {
+    async fn update_password(
+        &self,
+        ctx: &async_graphql::Context<'_>,
+        new_password: String,
+    ) -> async_graphql::Result<String> {
+        let pool = ctx.data::<sqlx::PgPool>()?;
+        let employee_id = ctx.data::<i32>()?;
+
+        update_password(new_password, pool, employee_id).await?;
+        Ok(String::from(""))
```

Comment:
I think an empty String is not a good successful message. I would return the employee object, of course without password and salt. The object could be easily extended to query related data (e.g. worktime and address).